vars/piperExecuteBin.groovy

```diff
@@ -25,6 +26,23 @@ void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, fai
 
         writeFile(file: ".pipeline/tmp/${metadataFile}", text: libraryResource(metadataFile))
 
+        // The default pipeline config, plus any custom default configs, have been
+        // extracted from the library resources into .pipeline/ by setupCommonPipelineEnvironment.groovy
+        // and fed into the DefaultValueCache. The 'default_pipeline_environment.yml' file
+        // is not reflected in the list returned by getCustomDefaults(), but has to be
+        // included in the --defaultConfig param. This makes sure that the same default configuration
+        // values are visible from the go side as from the Groovy side.
```

Comment:
Unnecessary comment. I think this should be from the code.

Should be clear that default_pipeline_environment is not in the list of CustomDefaults as it is not a custom default. It's is more a default default. :-)